Running-Mastodon/PgBouncer-guide.md

```diff
@@ -13,13 +13,18 @@ Note that you can check "PgHero" in the administration view to see how many Post
 Installing PgBouncer
 -----
 
-On Ubuntu:
+On Ubuntu/Debian:
 
     sudo apt install pgbouncer
 
-Restarting:
+Then:
 
-    sudo service pgbouncer restart
+    nano /etc/default/pgbouncer
+    START=1
```

Comment:
I was running Ubuntu 16.04? Maybe you were running a different version? I didn't need to do any of that…